sktalk/corpus/parsing/parser.py

```diff
@@ -1,9 +1,24 @@
 import abc
+import datetime
+from ..conversation import Conversation
 
 
 class Parser(abc.ABC):
     """Abstract parser class."""
 
     @abc.abstractmethod
-    def parse(self, file) -> "Conversation":  # noqa: F821
+    def parse(self, file) -> Conversation:
         return NotImplemented
+
+    @staticmethod
+    def _to_timestamp(time_ms):
```

Comment:
Looking at this one and the `ChaParser._split_time()` function, I think they would even justify a `timeutils` file so that all these functions are in one place.

Reply:
I think a `timeutils` file makes sense. Seems to fit also with the general preference we have (across all formats) to work exclusively with milliseconds, as documented in #12 with @liesenf. Essentially, we can't control what comes in as formats differ, but at all steps after import we only want to work with whole milliseconds across the board.

Reply:
I second that. To summarise:
- a utility function to covert between various timestamp formats and millisecond is useful
- the standard representation used across all of `scitkit-talk' is millisecond. This means some existing code might need to be edited

Reply:
See #30 for this issue! To be tackled outside of this PR :)